test/teiserver/protocols/spring/spring_auth_test.exs

```diff
@@ -461,12 +472,15 @@ CLIENTS test_room #{user.name}\n"
     # Rename with an invalid name
     _send_raw(socket, "RENAMEACCOUNT Y--Y\n")
     reply = _recv_raw(socket)
-    assert reply == "SERVERMSG Invalid characters in name (only a-z, A-Z, 0-9, [, ] allowed)\n"
+    assert String.starts_with?(reply, "SERVERMSG")
+    assert String.contains?(reply, "Invalid characters in name")
+    assert String.contains?(reply, "(only a-z, A-Z, 0-9, [, ] allowed)")
 
     # Rename with existng name
     _send_raw(socket, "RENAMEACCOUNT #{watcher_user.name}\n")
     reply = _recv_raw(socket)
-    assert reply == "SERVERMSG Username already taken\n"
+    assert String.starts_with?(reply, "SERVERMSG")
+    assert String.contains?(reply, "Username already taken")
```

Comment:
This is fine as it is. But you can leverage Elixir's pattern matching for something a little bit more direct:

```elixir
assert "SERVERMSG" <> rest = _recv_raw(socket)
assert String.contains?(rest, …)
```